### ISSUE TITLE
Document time-test usage

### ISSUE THREAD

**Issue by agjohnson** [ericholscher/time-test#8]
A brief summary of how this is configured would be helpful. I have some pages for latency checks and I'm trying to piece together how this is configured.

**Comment by humitos** [comment]
This is configured in the `monitoring` module from the `-ext` repository: https://github.com/readthedocs/readthedocs-ext/blob/c38d12292285bea393df95aa8fe33ca68b50b860/readthedocsext/monitoring/metrics/tasks.py#L165-L174

This data is sent to AWS CloudWatch and it checks for a threshold and decide whether or not send a PD alarm.

I'm not sure what's the explanation you are looking for. Can you expand on the details of this task?

**Comment by agjohnson** [comment]
This information is helpful. I'm looking to have this all documented somewhere -- a readme is fine. Because we get paged on this, we should share knowledge on how this is configured. I had to reverse engineer the whole set up when I got paged on the failures here. Docs describing how this is glued together would have helped greatly.

Also good to cover:

* time-test repo tests both queues, but it wasn't clear how (there are multiple branches set up and one triggers on build-large using readthedocs.yaml that inflicts conda usage, therefore build-large)

Basically, we should dump any information that someone new should have in order to understand what this repo is and how it works. I don't expect major future failures, the conda failure was rather special.
